buildLogic/plugins/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    `kotlin-dsl`
+}
+
+val kotlinVersion by rootProject.extra("1.9.0")
+val sonarqubeVersion by rootProject.extra("4.3.0.3225")
+
+dependencies {
+    listOf(
```

Comment:
Could these dependencies all come from the version catalog file?

Reply:
possibly yes, will havea look

Reply:
these can't access project version catalog for some reason :(